### PR TITLE
Setup alternative call for planner to use Google maps PhotoClient

### PR DESCRIPTION
**Pull request by rwangsc18**
## Description
To fix the missing photo issue, we want to experiment using Google maps photo api. Currently the planner uses a native http client that does send some customized request.

## Solution
* Define a new interface PhotoClient, insert it as a data member in Planner
* Add a flag "enable_maps_photo_client"
* Removed redundant unit tests

## Testing
- [x] Integration testing in local build
- [x] Removed new unit tests

## Checks
- [x] Have you removed commented code?
- [x] Have you used gofmt to format your code?

### CODE REVIEW

**Review by timwangmusic — APPROVED** [review]
lgtm, some minor comments

File: iowrappers/photos_client.go [hunk 1]
```diff
@@ -29,15 +20,36 @@ var isValidPhotoUrl = func(url string) bool {
...
-			return http.ErrUseLastResponse
-		},
+type PhotoURL string
+
+type PhotoClient interface {
+	GetPhotoURL(string) PhotoURL
+}
+
+// Use Google Map API
+type MapPhotoClient struct {
```
[the remark this answers is not in the document]
Comment (rwangsc18) [reply iowrappers/photos_client.go]: Done

File: planner/planner.go [hunk 2]
```diff
@@ -145,6 +145,17 @@ func (p *MyPlanner) Init(mapsClientApiKey string, redisURL *url.URL, redisStream
 	}
 	p.Configs = configs
 
+	// initialize photo client
+	var enable_maps_photo_client = false
+	if flag_enable_maps_photo_client, exists := p.Configs["server:plan_solver:enable_maps_photo_client"]; exists {
+		enable_maps_photo_client = flag_enable_maps_photo_client.(bool)
+		log.Debugln("load flag server:plan_solver:enable_maps_photo_client", enable_maps_photo_client)
```
[the remark this answers is not in the document]
Comment (rwangsc18) [reply planner/planner.go]: Done

**Comment by rwangsc18** [comment]
Thanks for the review!

**Pull request merged**